io/StacReader.cpp

```diff
@@ -387,8 +389,23 @@ std::string StacReader::extractDriverFromItem(const NL::json& asset) const
     return output;
 }
 
+// Create link relative to the Catalog/Item/Collection that is referencing it
+const std::string getPath(std::string srcPath, std::string linkPath)
+{
+    //Make absolute path of current item's directory, then create relative path from that
+
+    //Get driectory of src item
+    const std::string baseDir = FileUtils::getDirectory(srcPath);
+    if (FileUtils::isAbsolutePath(linkPath) ||
+            linkPath.find("://") != std::string::npos)
```

Comment:
Shouldn't this check actually go in `FileUtils::isAbsolutePath`? I'm confused why you are testing for a protocol.

Reply:
Because `isAbsolutePath` is already an established check, and I thought it would be better to do any more explicit checks on top of it rather than inside of it. Though as far as I'm aware, it should be true that anything with a protocol would be an absolute path, so I could move it inside of the `isAbsolutePath` check.